### ISSUE TITLE
Small view: Asset's info and extended balance

### ISSUE THREAD

**Issue by Sergei-Beam** [BeamMW/web-wallet#180]
moved from https://github.com/BeamMW/web-wallet/issues/56

4. Asset's info and extended balance
Asset's info screen should contain 2 new tabs:
- Balance https://zpl.io/ag3dEGZ
- Asset info https://zpl.io/ad35oM7

4.1 BALANCE tab
**Balance** tab shows all information about asset's coins.
Here we should have two spaces (expanded by default):
AVAILABLE
LOCKED

4.1.1 AVAILABLE contains the information below:
- available
- regular (amount which is received as regular coins)
- **maximum anonymity** (instead of max privacy as on design) (amount which is received as maximum anonymity coins)
**Blocked by** - https://github.com/BeamMW/beam/issues/1870

4.1.2 LOCKED contains the information below:
- locked (amount which is locked for a while)
- maturing
- change
-  **maximum anonymity** (instead of max privacy as on design) (amount which is received as maximum anonymity coins)

4.2 ASSET INFO tab
Asset's info should contain all related information from asset metadata.
It should be rendered dynamically, all empty fields should be hidden

Web site icon should contain the URL from  metadata and redirect to it after clicking if it is
Description paper icon should redirect to explorer with the opened asset info page

**Comment by razoorka** [comment]
On hold because of lack of information in API.
